src/App.tsx

```diff
@@ -1,16 +1,22 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setNewMovies] = useState(moviesFromServer);
```

Comment:
```suggestion
  const [movies, setMovies] = useState(moviesFromServer);
```
Name state like this [anyPossibleName, setAnyPossibleName]. State name should be identical for just state and for setState.